rfcs/protocols/asset-transfer/generic.md

```diff
@@ -6,13 +6,14 @@
 # Asset Transfer Protocol
 
 - RFC: 02-008
-- Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Authors: Krishnasuri Narayanam, Sandeep Nishad, Venkatraman Ramakrishna, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Jun-2021
+- Since: 21-Oct-2022
 
 ## Summary
 
-A protocol to transfer an asset from one shared ledger to another by atomically burning it in the source ledger and recreating it in the destination ledger.
+Asset transfer protocol in Weaver allows transferring an asset from one shared ledger to another by burning it in the source ledger and minting it in the destination ledger in an atomic manner.
+- Minting of the asset in the destination ledger should happen within an agreed-upon period. Otherwise, the asset is minted in the source ledger after the time period elapses.
```

Comment:
I'm wondering if we should be using the word "mint" or "create". I think "mint" is appropriate for coins, and perhaps even for fungible tokens in general, but "create" is probably more general. @sanvenDev @dhinakaran2705 your opinions?

Reply:
I agree `create` can be more general.